tests/test_a11y.py

```diff
@@ -38,8 +38,9 @@ def filter_ignored_violations(violations, url_pathname):
     ]:
         filtered = []
         for violation in violations:
-            # TODO: eventually fix this rule violation. See
-            # https://github.com/pydata/pydata-sphinx-theme/issues/1479.
+            # TODO: remove this exclusion once the following update to Axe is
```

Comment:
This change is unrelated to the rest of the PR, but I thought I would go ahead make this comment more up to date